bazel_tools/distribute_jar_cleanup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+SRC=$1
+OUT=$2
+NOTICE=$3
+LICENSE=$4
+LICENSE_EE=$5
+IS_EE=$6
+
+cp $SRC $OUT
+chmod +w $OUT
+zip -dq $OUT /LICENSE* /META-INF/LICENSE* /NOTICE* /META-INF/NOTICE*
+zip -uq $OUT $NOTICE
+
+if [[ "$IS_EE" -eq 1 ]]; then
+  # LICENSE_EE has the wrong file name/directory, so we copy to a temp dir and 
+  TMP_LICENSE_EE_DIR=$(mktemp -d)
+  cp $LICENSE_EE $TMP_LICENSE_EE_DIR/LICENSE
+  zip -ujq $OUT $TMP_LICENSE_EE_DIR/LICENSE
```

Comment:
```suggestion
  cp $LICENSE_EE $TMP_LICENSE_EE_DIR/LICENSE.txt
  zip -ujq $OUT $TMP_LICENSE_EE_DIR/LICENSE.txt
```
I'm not sure how important the exact name is here but this is the name legal originally asked for.

Reply:
Actually the change is a bit more involved if we want the apache file name to match. 🤔

Reply:
Can we standardize on `LICENSE.txt`? That's what we have in tarballs and Docker images. The above becomes (untested):
```
TMP_LICENSE_DIR=$(mktemp -d)
if [[ "$IS_EE" -eq 1 ]]; then 
  cp $LICENSE_EE $TMP_LICENSE_DIR/LICENSE.txt
else
  cp $LICENSE $TMP_LICENSE_DIR/LICENSE.txt
fi
zip -ujq $OUT $TMP_LICENSE_DIR/LICENSE.txt
rm -rf $TMP_LICENSE_DIR
```

Reply:
sure!

Reply:
Should notices also be .txt?

Reply:
I'm not aware of any specific requirement on that one so let's keep it as NOTICES.